index.js

```diff
@@ -0,0 +1,37 @@
+window.addEventListener('DOMContentLoaded', () => {
+
+    const popupImages = document.querySelectorAll('.featured-img__style');
+    const popup = document.querySelector('.popup');
+
+    popupImages.forEach((img, index) => {
+        img.addEventListener('click', (img) => {
+            document.querySelector('.popup-img').src = img.target.src;
+        });
+        img.addEventListener('click', openPopup);
+    });
+
+    function openPopup() {
+        popup.classList.add('show-popup');

```

Comment:
Rework it with classList.toggle. You can rewrite it with only one class modifier.

Reply:
Thanks for this recommendation
I use two different functions, I did not understand a little how to do this in one function. If this is a gross error, then of course I will fix it.